packages/zoe/src/clientSupport/index.js

```diff
@@ -1 +1 @@
-export { makeGetOfferHandle, makeGetInstanceHandle } from './offerHandle';
+export { makeGetInviteHandle, makeGetInstanceHandle } from './inviteHandle';
```

Comment:
Because it's obtaining the handle in question from the invite extent. We agreed that the invite extent will only carry the inviteHandle, not the offerHandle.

Reply:
Right, but no one should need to get an inviteHandle now, if they get the offerHandle as the result of E(zoe).offer. I think we can remove it.

Reply:
Tried. Problem: The one place we use the gotten inviteHandle is in test-simpleExchange

```js
  // TODO(msm): Capture the offerHandle here and use it in the test below.
  await zoe.offer(aliceInvite, aliceSale1OrderProposal, alicePayments);

  const expected = [{ Price: 4 }, { Asset: 3 }];

  t.deepEquals(publicAPI.getOffer(aliceInviteHandle), expected);
```
We actually want an offerHandle here, which further supports getting of it.
However, when we do what the above TODO suggests, we find out that the offerHandle is for `'not an active offer'`. Doh!

Reply:
It seems like this test should be testing an offer that doesn't have a match - that offer will stick around and we can get information on it

Reply:
Gone. Done.